docs/changelog.rst

```diff
@@ -2,6 +2,9 @@
 Changelog
 =========
 
+* :release:`7.0.1 <2020-06-02>`
+* :bug:`255 major` Update desired_mode method in LiquidHandle to allow for cases where mode_param is empty
```

Comment:
if we're releasing right away with a bugfix, we can use `255` without the major flag

Reply:
updated